librz/core/cmd/cmd_print.c

```diff
@@ -3517,6 +3474,19 @@ RZ_IPI RzCmdStatus rz_cmd_disassembly_function_handler(RzCore *core, int argc, c
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_print_current_block_json_handler(RzCore *core, int argc, const char **argv) {
+	rz_return_val_if_fail(core, RZ_CMD_STATUS_ERROR);
+	bool enable_color = rz_config_get_i(core->config, "escr.color") != 0;
+	char *res = rz_print_json_indent((const char *)core->block, enable_color, "  ", NULL);
+	if (RZ_STR_ISEMPTY(res)) {
+		rz_cons_printf("Couldn't find a JSON string.\n");
```

Comment:
Shouldn't this return RZ_CMD_STATUS_ERROR?

Reply:
Also maybe better using `RZ_LOG_ERROR()` here?

Reply:
yeah, throw an error saying invalid json (empty string).